webpack.config.js

```diff
@@ -106,5 +107,15 @@ module.exports = {
       });
       return middlewares;
     }
-  }
+  },
+  optimization: {
+    minimize: true,
+    minimizer: [
+      new TerserPlugin({
```

Comment:
nit: In the webpack website: `**Disclaimer: TerserWebpackPlugin is a third-party package maintained by community members, it potentially does not have the same support, security policy or license as webpack, and it is not maintained by webpack.**`, do we have any other options? Also, do we need to add it into dev dependency by `npm install`?